sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorConnection.java

```diff
@@ -95,18 +99,59 @@ public ReactorConnection(String connectionId, ConnectionOptions connectionOption
         this.handler = handlerProvider.createConnectionHandler(connectionId, connectionOptions);
 
         this.retryPolicy = RetryUtil.getRetryPolicy(connectionOptions.getRetry());
+        this.operationTimeout = connectionOptions.getRetry().getTryTimeout();
```

Comment:
`connectionOptions.getRetry().getTryTimeout()`  : This is documented generically to user as `maximum duration to wait for completion of a single attempt`, we might have to expose this to user as a separate configuration in future.